src/test/java/com/jcabi/github/RtHooksITCase.java

```diff
@@ -146,9 +148,22 @@ private static Repos repos() {
      * @throws IOException If there is any I/O problem
      */
     private static Hook createHook(final Repo repo) throws IOException {
+        final ConcurrentHashMap<String, String> config =
```

Comment:
@pinaf That would cause [PMD UseConcurrentHashMap](http://pmd.sourceforge.net/pmd-4.3.0/rules/controversial.html) qulice error.